src/pages/api/sendEmail.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+
+import { firestore } from '@/lib/temp/firebase/admin/init'
+
+interface Message {
+  message: {
+    subject: string
+    text: string
+    html: string
+  }
+}
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
This should be a firebase functions. I can create a new issue to convert this function into a firebase function

Reply:
Yeah I think do this as a separate issue